### ISSUE TITLE
Can't build from zip version

### ISSUE THREAD

**Issue by ekner** [openSUSE/hwinfo#56]
I am able to build the package when I download the package with git, but not when I download the zip file, or the latest release (21.51). I get the following errors when running make:

cat: VERSION: No such file or directory
cat: /home/gustav/Downloads/hwinfo-master/VERSION: No such file or directory

Also, perhaps there is a good idea to mention that libx86emu id a dependency somewhere in the readme.

**Comment by BKPepe** [comment]
I downloaded tarball from Github with version 21.67 and have exactly the same issue.

```
cat: ../../VERSION: No such file or directory
cut: ../../VERSION: No such file or directory
```

```
no git repo
cat: VERSION: No such file or directory
```

`*** No rule to make target '../../VERSION', needed by 'version.h'.  Stop.`

**Comment by wfeldt** [comment]
Yes, this does unfortunately not work. I'll give it some thought.
